pom.xml

```diff
@@ -32,6 +46,44 @@
   </dependencyManagement>
 
   <dependencies>
+    <dependency>
+      <groupId>net.adoptopenjdk.api</groupId>
+      <artifactId>adoptopenjdk-api-v3-models</artifactId>
+      <version>3.0.0-SNAPSHOT</version>
+    </dependency>
+    <dependency>
+      <groupId>org.jetbrains.kotlin</groupId>
+      <artifactId>kotlin-reflect</artifactId>
```

Comment:
Why do we depend on Kolin?

Reply:
jackson-module-kotlin fails to serialize json responses without it, but works when it's included (openjdk-api-v3-models are written in Kotlin)...